lib/tasks/zync.rake

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+namespace :zync do
+  namespace :resync do
+    def each_with_progress(task_desc, scope)
```

Comment:
`task_desc` is not used anywhere?

Reply:
Right, forgot that. I has a header message printed to STDOUT before.

Reply:
Fixed